docs/source/release/v6.0.0/diffraction.rst

```diff
@@ -5,115 +5,124 @@ Diffraction Changes
 .. contents:: Table of Contents
    :local:
 
-.. warning:: **Developers:** Sort changes under appropriate heading
-    putting new features at the top of the section, followed by
-    improvements, followed by bug fixes.
-
-New features
-------------
-
 Powder Diffraction
 ------------------
-New features
+
+New Algorithms
+##############
+
+- :ref:`PaalmanPingsAbsorptionCorrection <algm-PaalmanPingsAbsorptionCorrection>` uses a numerical integration method to calculate attenuation factors for all Paalman Pings terms
+- :ref:`LeadPressureCalc <algm-LeadPressureCalc>` to calculate the pressure of a sample given the d-Spacing of the Lead (111) peak and the sample temperature
```

Comment:
Someone from the SNS dropped a line from the release notes back in December for some reason. Please can you reinsert this here. It belongs next to the comment about the lead pressure calc:
- Added a new :ref:`PEARLTransfit <algm-PEARLTransfit>` Algorithm for PEARL that uses a TransVoigt function to determine the temperature of a given sample

Also it's perhaps worth moving the LeadPressureCalc and PEARLTransfit items to the top of the "New Algorithms" section because PaalmanPingsAbsorptionCorrection line and all the Corelli lines are all SNS. So nice to keep them together.